internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -227,10 +258,10 @@ func (opts *Opts) askClusterOptions() error {
 
 	clusterName := opts.ClusterName
 	if clusterName == "" {
-		message := "Insert the cluster name"
+		message := ""
 		clusterName = opts.newClusterName()
 		if clusterName != "" {
-			message = fmt.Sprintf("Insert the cluster name [Press Enter to use the auto-generated name '%s']", clusterName)
+			message = fmt.Sprintf("[Press Enter to use the auto-generated name '%s']:", clusterName)
```

Comment:
and now seeing this you add the colon again so double colon

Reply:
consider that the format I suggested for the cluster `Cluster Name%s:` account for the message to be an empty string, which may be the case from what I see in your code so all this should follow. a similar approach